src/js/modules/RoutingApp.js

```diff
@@ -181,12 +180,6 @@ class RoutingApp extends Component {
                 key={"latest"}
                 component={LatestPage}
               />
-              <Route
-                exact
-                path={"/search"}
-                key={"search"}
-                component={SearchPage}
-              />
               <Route

```

Comment:
I deleted all Search functionality to keep this branch pure. The search actions/components will be in @JasonLin43212's next PR.